sql/hive/src/main/scala/org/apache/spark/sql/hive/HiveQl.scala

```diff
@@ -557,7 +557,6 @@ https://cwiki.apache.org/confluence/display/Hive/Enhanced+Aggregation%2C+Cube%2C
             "TOK_TABLEPROPERTIES"),
           children)
       val (db, tableName) = extractDbNameTableName(tableNameParts)
-
       CreateTableAsSelect(db, tableName, nodeToPlan(query), allowExisting != None, Some(node))
```

Comment:
Is it the only place where we create a `CreateTableAsSelect`? If so, can we get rid of the `Option` and just pass the `node`?